db/migrate/20110616080444_add_look_up_key_id_to_puppet_class.rb

```diff
@@ -11,7 +11,6 @@ def up
     add_column :lookup_keys, :validator_type, :string
     add_column :lookup_keys, :validator_rule, :string
     rename_column :lookup_values, :priority, :match
-    add_index :lookup_values, :match
```

Comment:
It's on the commit message, basically renaming the index is done automatically by Rails 4, but not by Rails 3.

```
An index is added previously on lookup_values :priority, and on Rails 4
rename_column changes 'priority' to 'match' already changes the name of
the index.
```

The original index on lookup_values priority was added in db/migrate/20091219132839_create_lookup_values.rb